### PR TITLE
chore: vec-290 change basic, quote, and prism example default AVS dat…

### PR DESCRIPTION
**Pull request by dwelch-spike**
…a and index namespaces to avs-data and avs-index. Also adds a readme note about the use of separate namespaces for AVS to each example.

### CODE REVIEW

File: basic-search/README.md [hunk 1]
```diff
@@ -24,7 +24,26 @@ source .venv/bin/activate
...
+
+[!NOTE]
+It is best practice to store AVS index and record data in separate namespaces.
+By default this application stores its AVS index in the "avs-index" namespace, and AVS records in "avs-data".
+If your Aerospike database configuration does not define these namespaces you will see an error.
+You may change the --namespace and --index-namespace to other values, like the default Aerospike "test" namespace, to use other namespaces.
+
+| Command Line Flag        | Default            | Description                                                        |
+|-----------------------------|--------------------|-----------------------------------------------------------------|
+| --host               | localhost          | AVS server seed host                                                   |
```
Comment (jdogmcsteezy) [inline basic-search/README.md:40]: It is only a seed when load balancer is false. I think it is probably fine as is

Comment (jdogmcsteezy) [inline basic-search/README.md:40]:
```suggestion
| --host               | localhost          | AVS host used for initial connection                                                   |
```

**Review by jdogmcsteezy — COMMENTED** [review]
No changes are needed to any docker compose or config files? I am wondering if quote/prism worked prior to these changes if they still relied on the test namespace?

**Comment by dwelch-spike** [comment]
> No changes are needed to any docker compose or config files? I am wondering if quote/prism worked prior to these changes if they still relied on the test namespace?

They work if your deployment uses the test namespace. The docker compose files set the environment variables so that these examples use the ave-data and avs-index separate namespaces, so they work either way. That's what Adam wanted when I spoke with him originally, "test" by default and separate namespaces with the docker composes.

File: basic-search/README.md (hunk 1 above)
Comment (dwelch-spike) [reply basic-search/README.md]: Used your suggestion.

**Comment by dwelch-spike** [comment]
@jdogmcsteezy I removed the environment variables for separate namespaces from the example dockerfiles as they are now defaults in the apps.

**Review by jdogmcsteezy — APPROVED** [review]
assuming it is tested, LGTM

**Comment by dwelch-spike** [comment]
These are tested locally. I'll merge this after the conflicts are fixed.

**Pull request merged**